app/controllers/downloader.js

```diff
@@ -65,6 +69,10 @@ export default Ember.Controller.extend({
     };
   },
 
+  __episodeURL (episode) {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).